### PR TITLE
Use a null cache adapter for the status controller

### PR DESCRIPTION
**Pull request by Tasiobg**
The status controller should not connect to redis and setting the TTL to CacheInterface::NONE is not avoiding the connection, this PR updates the dependency injection to use a null cache adapter on the status controller.

One of the side effects of the previous solution is to display a lot of cache miss on the CloudWatch Dashboad because the health check runs at least once every second even when we set that to run every 5 minutes

### CODE REVIEW

**Comment by programmes-developers** [comment]
Can one of the admins verify this patch?

File: src/Controller/StatusController.php [hunk 1]
```diff
@@ -16,22 +19,34 @@
...
+    }
+
     public function __invoke(
         Request $request,
         Connection $dbalConnection,
-        ProgrammesService $programmesService,
-        BroadcastsService $broadcastsService,
-        VersionsService $versionsService,
-        SegmentEventsService $segmentEventsService
+        EntityManagerInterface $entityManager
```
Comment (BjornTwachtmann) [inline src/Controller/StatusController.php]: I don't think this is used any more, is it?

**Comment by BjornTwachtmann** [comment]
Other than that one tiny thing, fine, merge it.

File: src/Controller/StatusController.php (hunk 1 above)
Comment (Tasiobg) [reply src/Controller/StatusController.php]: nop, that was for the old aproach

**Pull request merged**